analysis/README.md

```diff
@@ -101,9 +101,9 @@ Rom generation generates a single ROM for each virtual machine using the followi
 - Find the maximum number of inputs among all functions. Introduce as many input registers. Do the same for outputs, introducing output registers
 - Replace references to the function arguments by references to these input registers.
 - Pad all return statements with zeroes up to the number of output registers.
-- Inline all functions, adding a label before each one.
+- Inline all functions arbitrarily ordered by name, adding a label before each one.
```

Comment:
```suggestion
- Inline all function bodies, arbitrarily sorted by name, adding a label before each one.
```
Otherwise it's not clear to me what "inline all functions" mean